apps/dashboard/config/configuration_singleton.rb

```diff
@@ -267,11 +267,16 @@ def config
   end
 
   def read_config
-    f = Configuration.config_file
-    YAML.safe_load(File.open(f.to_s)).symbolize_keys
-  rescue => e
-    Rails.logger.error("Can't read or parse #{Configuration.config_file} because of error #{e}")
-    {}
+    files = Pathname.glob([config_directory.join("*.{yml,yaml}"), config_directory.join("*.{yml,yaml}.erb")])
```

Comment:
You can do

    config_directory.join("*.{yml,yaml,yml.erb,yaml.erb}

Reply:
Done.